components/web/nginx/Makefile

```diff
@@ -65,8 +68,9 @@ CONFIGURE_OPTIONS += --with-mail
 CONFIGURE_OPTIONS += --with-mail_ssl_module
 CONFIGURE_OPTIONS += --with-ipv6
 CONFIGURE_OPTIONS += --with-threads
+CONFIGURE_OPTIONS += --add-module=$(@D)/ngx-fancyindex
 
-COMPONENT_PRE_CONFIGURE_ACTION = ($(CLONEY) $(SOURCE_DIR) $(@D))
+COMPONENT_PRE_CONFIGURE_ACTION = ($(CLONEY) $(SOURCE_DIR) $(@D); git clone https://github.com/aperezdc/ngx-fancyindex.git $(@D)/ngx-fancyindex; cd $(@D)/ngx-fancyindex; git checkout v$(NGX_FANCYINDEX_VERSION))
 COMPONENT_POST_INSTALL_ACTION+= \
```

Comment:
This works but looks ugly. I suggest to split it into separate lines, eg.
COMPONENT_PRE_CONFIGURE_ACTION = ($(CLONEY) $(SOURCE_DIR) $(@D) && \
 git clone https://github.com/aperezdc/ngx-fancyindex.git $(@D)/ngx-fancyindex && \
  cd $(@D)/ngx-fancyindex && \
   git checkout v$(NGX_FANCYINDEX_VERSION))

Reply:
Done